server/routes/index.ts

```diff
@@ -2,13 +2,15 @@ import { type RequestHandler, Router } from 'express'
 
 import asyncMiddleware from '../middleware/asyncMiddleware'
 import type { Services } from '../services'
+import config from '../config'
 
 export default function routes({ supportedPrisonsService }: Services): Router {
   const router = Router()
   const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  const { oneLoginLink } = config
 
   get('/', (req, res, next) => {
-    res.render('pages/index', { user: req.user })
+    res.render('pages/index', { user: req.user, oneLoginLink })
```

Comment:
I think passing `user` is a hangover from displaying all that debug info so we can remove it at some point...

Passing `oneLoginLink` works here but we'll probably want to eventually move it so it's always available - otherwise we'll have to pass it on every page. For example, could go here (https://github.com/ministryofjustice/hmpps-book-a-prison-visit-ui/blob/c6a863c0152e1c581876d85879ea971a15e945b4/server/utils/nunjucksSetup.ts#L14-L17) then it will always be available in `res.locals`. Fine for this PR though.